maintenance/tasks.cron.txt

```diff
@@ -0,0 +1 @@
+0 5 * * * /var/www/colouringlondon/maintenance/extract_data/extract_data.py
```

Comment:
1. AFAIK the path needs to job script needs to be absolute so I put the standard path to the project on our servers
2. This file could be imported with `cat tasks.cron.txt | crontab -e -` but that would override any other jobs that are not in this file. The solution could be:
a) version control all cron tasks we want to run on the server in the `tasks` file (more comprehensive but might lead to exposing some details that are specific only to our server set-up, like the back-up schedule)
b) do not store cron schedules in version control (but then the schedules, which are sometimes tied to the proper functioning of the website, could get lost / accidentally overwritten etc)

Reply:
1. Looks fine
2. Do you think we should move to version-controlled config in a private repo? I.e. option a) but avoiding exposing unnecessary detail.

Reply:
I would say versioning the configuration but separately from the source would be the way to go. Can you please create the private repo then?